jenkins/deployment.groovy

```diff
@@ -76,120 +76,95 @@ node('master') {
+            AWS_ACCESS_KEY_CREDENTIALS_ID = params.AWS_CUSTOM_ACCESS_KEY_ID
+            AWS_SECRET_CREDENTIALS_ID = params.AWS_CUSTOM_SECRET_ACCESS_KEY
           }
-        } catch(e) {
-          currentBuild.result = 'FAILED'
-          wireSend secret: "$jenkinsbot_secret", message: "**Setting environment variables failed for ${version}** see: ${JOB_URL}"
-          throw e
-        }
 
         try {
-          sh "jenkins/ts-node.sh ./bin/deploy-tools/s3-cli.ts --bucket \"${params.S3_BUCKET}\" --s3path \"${S3_PATH}\" --key-id \"${AWS_ACCESS_KEY_ID}\" --secret-key \"${AWS_SECRET_ACCESS_KEY}\" --wrapper-build \"${WRAPPER_BUILD}\" --path \"${SEARCH_PATH}\" ${DRY_RUN}"
+          withCredentials([
+            string(credentialsId: AWS_ACCESS_KEY_CREDENTIALS_ID, variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: AWS_SECRET_CREDENTIALS_ID, variable: 'AWS_SECRET_ACCESS_KEY')
+          ]) {
+            sh "jenkins/ts-node.sh ./bin/deploy-tools/s3-cli.ts --bucket \"${S3_BUCKET}\" --s3path \"${S3_PATH}\" --key-id \"${AWS_ACCESS_KEY_ID}\" --secret-key \"${AWS_SECRET_ACCESS_KEY}\" --wrapper-build \"${WRAPPER_BUILD}\" --path \"${SEARCH_PATH}\" ${DRY_RUN}"
+          }
         } catch(e) {
           currentBuild.result = 'FAILED'
           wireSend secret: "$jenkinsbot_secret", message: "**Deploying to S3 failed for ${version}** see: ${JOB_URL}"
           throw e
         }
       } else if (projectName.contains('macOS')) {
         try {
-          def MACOS_HOCKEY_ID = ''
-          def MACOS_HOCKEY_TOKEN = ''
+          def appName
+          def distributionGroups
 
           if (params.Release.equals('Production')) {
-            withCredentials([
-              string(credentialsId: 'MACOS_MAS_HOCKEY_ID', variable: 'MACOS_MAS_HOCKEY_ID'),
-              string(credentialsId: 'MACOS_MAS_HOCKEY_TOKEN', variable: 'MACOS_MAS_HOCKEY_TOKEN')
-            ]) {
-              MACOS_HOCKEY_ID = env.MACOS_MAS_HOCKEY_ID
-              MACOS_HOCKEY_TOKEN = env.MACOS_MAS_HOCKEY_TOKEN
-            }
+            appName = 'Wire-macOS-Production'
           } else if (params.Release.equals('Custom')) {
-            withCredentials([
-              string(credentialsId: "${params.MACOS_CUSTOM_HOCKEY_ID}", variable: 'MACOS_CUSTOM_HOCKEY_ID'),
-              string(credentialsId: "${params.MACOS_CUSTOM_HOCKEY_TOKEN}", variable: 'MACOS_CUSTOM_HOCKEY_TOKEN')
-            ]) {
-              MACOS_HOCKEY_ID = env.MACOS_CUSTOM_HOCKEY_ID
-              MACOS_HOCKEY_TOKEN = env.MACOS_CUSTOM_HOCKEY_TOKEN
-            }
-          } else {
-            withCredentials([
-              string(credentialsId: 'MACOS_HOCKEY_ID', variable: 'MACOS_HOCKEY_ID'),
-              string(credentialsId: 'MACOS_HOCKEY_TOKEN', variable: 'MACOS_HOCKEY_TOKEN')
-            ]) {
-              MACOS_HOCKEY_ID = env.MACOS_HOCKEY_ID
-              MACOS_HOCKEY_TOKEN = env.MACOS_HOCKEY_TOKEN
-            }
+            appName = 'Wire by XY - macOS'
```

Comment:
Does this need to be a valid App Center key?

Reply:
I now throw an error (the script will stop directly) so that we do not forget to add real values

Reply:
Please re-check again